src/app/public/modules/key-info/key-info.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
```

Comment:
Should these imports be on separate lines?

Reply:
I'm hoping to not change the original source too much on this initial port in case something changes in the master branch before we end up releasing this. I definitely want to refactor the code style for all libraries, soon.